### DISCUSSION TITLE
Group Assignment #2 - Memo

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/22S-ASIAAM-191A#13] [Assignments]
## Due 4/20

Have one person comment on this post to submit the link to your [group assignment #2](https://albertkun.github.io/22S-ASIAAM-191A/assignments/week2/group_assignment/) memo with group name in this thread.

### For example

Awesome Group - [https://docs.google.com/document/d/1OWUUxebPNsqw4K97gznvTkGLEr6QDAbXWqX2BGvernI/copy](https://docs.google.com/document/d/1OWUUxebPNsqw4K97gznvTkGLEr6QDAbXWqX2BGvernI/copy)

**Comment by XiongHer** [comment]
Ngoc, Willa, Xiong - https://docs.google.com/document/d/1oNlmaNkFBhE87tA2xJ77YnEgUnRHy29786SlHvXmvQw/edit?usp=sharing

**Comment by kericsp** [comment]
Keri, Kristen, Emma - https://docs.google.com/document/d/1luEGoedbzhedIfqYMA7nWkhUxx3iNjDuX2HC2Sblv8w/edit?usp=sharing

**Comment by joonyloony** [comment]
Jenny, Ananya https://docs.google.com/document/d/1F7eX4HvRmtT9miAq7_OGafAOFyV_5kV4Jy7CEx_vau4/edit?usp=sharing

**Comment by ashley-lu** [comment]
Ashley, Haven, Shubham: https://docs.google.com/document/d/1ixQYmUilUsEuYVDs8aOE7snv9UvFk105X7b6BBjRmUo/edit?usp=sharing

**Comment by kaittli** [comment]
Kaitlyn, Antony: https://docs.google.com/document/d/1cnGiWugKU826Ou9nGNnpixDIuSVHKcpYEq9DLA8szbs/edit?usp=sharing

**Comment by daikis12** [comment]
Lindsey, Angeline, Daiki: https://docs.google.com/document/d/1rTA8Ie0XUcuqYRIij91iTfYFv1I48S7zOmdjDv41UCs/edit#

**Comment by codingdianat** [comment]
Diana Tran, Steven Tran: https://docs.google.com/document/d/1ntyzKAuOQ5EyIGzjUt9FMpkmcq12B52i-Wkx9gCwVVY/edit?usp=sharing

**Comment by NishuKrish** [comment]
Nishaanth Krishnan, Ivy Tang: https://docs.google.com/document/d/1kZpRNve7cEyXLhCrZJw2k9Qs9fWhWY3x6NW1lwuPXxQ/edit?usp=sharing

**Comment by spolcharoen** [comment]
Sine, Lillan, Alexis - https://docs.google.com/document/d/1SDURCaoLo6TZAHLAzLEsBFkwoq1QmT54mR6YNdFi1fw/edit?usp=sharing
